src/router/AppRouter.jsx

```diff
@@ -1,14 +1,17 @@
 import { RouterProvider, createBrowserRouter } from 'react-router-dom'
 import { HomePage, ManagePage } from '../pages'
+import { Error404 } from '../pages/Error404'
 
 const router = createBrowserRouter([
    {
       path: '/',
-      element: <HomePage />
+      element: <HomePage />,
+      errorElement: <Error404 />
```

Comment:
the errorElement is shown when exceptions are thrown, if an error occurs in the homepage it will redirect the user to the errorElement component, this is not the behavior we want.

Instead similar to the old versions of react router we can use a wild card * to redirect the user.